x/gamm/spec/README.md

```diff
@@ -968,6 +950,48 @@ osmosisd tx gamm swap-exact-amount-out 1000000ibc/27394FB092D2ECCD56123C74F36E4C
 ```
 
 ## Other resources
-* [Creating a liquidity bootstrapping pool](./client/docs/create-lbp-pool.md)
-* [Creating a pool with a pool file](./client/docs/create-pool.md)
 
+- [Creating a liquidity bootstrapping pool](./client/docs/create-lbp-pool.md)
+- [Creating a pool with a pool file](./client/docs/create-pool.md)
```

Comment:
Looks like your IDE has the asterisk -> dash auto formatting too – should probably keep them as asterisks for now to be consistent with the rest of our markdown files but this is something I'm dealing with too

Reply:
I mean we can switch, I don't care that much. Just don't want a continual back and forth between the two.